src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,36 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) {
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+            byte[] encrypted = cipher.doFinal(value.getBytes());
+            return StringHelper.byteArrayToHexString(encrypted);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    @Test
+    public void testDefaultAesCipherParams() throws InvalidParametersException {
+        String msg1 = "AAAAAA";
+        //128 bits and more
+        String msg2 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
+        String msg3 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCCAAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
```

Comment:
Sorry should be plaintext, not ciphertext.

For the comment, what about
```java
    //Using plaintext > 128 bits to check for any differences operations
```

The plaintextLength136 is actually 144 bits long. Suggest doing this so its easier to read.
```java
String plaintextLength136 = plaintextLength128 + "...";
```